<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="Input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>521</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="AddButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>6</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Polynomial</string>
    </property>
   </widget>
   <widget class="QLabel" name="StatusLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="BaseView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>651</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ActionStatus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>631</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="get_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Index</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="get_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Degree</string>
    </property>
   </widget>
   <widget class="QPushButton" name="get_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Coeff</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="add_1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>440</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Index</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>500</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="add_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>470</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Index 2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mul_1">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>440</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Index</string>
    </property>
   </widget>
   <widget class="QPushButton" name="roots_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>500</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Roots</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mul_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>500</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Multiply</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mul_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>470</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Index 2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="roots_1">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>440</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Index</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="der_1">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>440</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Index</string>
    </property>
   </widget>
   <widget class="QPushButton" name="der_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>500</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Derivative</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="der_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>470</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>N</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="del_1">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>440</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Index</string>
    </property>
   </widget>
   <widget class="QPushButton" name="del_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>500</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QLabel" name="Error">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="icons.qrc">:/:/Polynomials/Error.bmp/static/error.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Success">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="icons.qrc">:/:/Polynomials/Ok.bmp/static/success.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>674</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_from_file"/>
    <addaction name="actionSave_to_file"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_from_file">
   <property name="text">
    <string>Load from file</string>
   </property>
  </action>
  <action name="actionSave_to_file">
   <property name="text">
    <string>Save to file</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
